sentry-android-core/api/sentry-android-core.api

```diff
@@ -1,6 +1,7 @@
 public final class io/sentry/android/core/ActivityFramesTracker {
 	public fun <init> (Lio/sentry/android/core/LoadClass;)V
 	public fun <init> (Lio/sentry/android/core/LoadClass;Lio/sentry/ILogger;)V
+	public fun <init> (Lio/sentry/android/core/LoadClass;Lio/sentry/ILogger;Lio/sentry/android/core/MainLooperHandler;)V
```

Comment:
@marandaneto I added the handler argument to both the test and the non-test ctor, this now changes the .api file - what do you think, should the handler be an argument for the `@TestOnly` constructor or is it fine this way?

Reply:
It's fine IMO.